tests/console/x_vt.pm

```diff
@@ -9,7 +9,6 @@
 # without any warranty.
 
 # Summary: Check for correct tty used by X
-# - Check if X is running
```

Comment:
@emiura I see that you added this line in 34b55985d. It seems this was adding to confusing what is the test scope of the the module. I propose the deletion of this line again as it's simply not the test goal.

Reply:
@okurz I don't have anything agains deleting this line, however, couldn't we just replace by the correct description of what the test does?

Reply:
Well, the already existing summary line explains exactly what the test does, which is not much. Do you think we need to extend on "Check for correct tty used by X"?